[Bug]: Error: layers.xxx 'undefined' value invalid

### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/sbb-angular/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/sbb-angular/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/1) for an issue that matches the one I want to file, without success.

### Bug type

Other

### Which version are you using

16.8.0

### Operating system

Windows 10

### Browser / Browser version

Microsoft Edge

### Input mode

None

### Additional settings

_No response_

### What happened?

After updating the journey-maps library to 16.8.0, i see errors in the browser console regarding "undefined", e.g.:

`Error: layers.level-street-lvl.filter[1][2]: 'undefined' value invalid. Use null instead.`

The errors are also visible when opening the console on the SBB Angular Journey Maps Example page:
`https://sbb-angular.app.sbb.ch/journey-maps/components/angular/examples`

The errors do not occur with v16.7.1

### Steps to reproduce the issue

1. Update journey maps to 16.8.0
2. Check console

### Relevant log output

```shell
angular-WCVVIDOZ.js:1  Error: layers.level-street-k-lvl.filter[1][2]: 'undefined' value invalid. Use null instead.
    at Object.J4 [as emitValidationErrors] (angular-WCVVIDOZ.js:2:6906)
    at St (angular-WCVVIDOZ.js:2:346068)
    at Pi._validate (angular-WCVVIDOZ.js:2:368867)
    at Pi.setFilter (angular-WCVVIDOZ.js:2:362252)
    at Map.setFilter (angular-WCVVIDOZ.js:576:166014)
    at angular-WCVVIDOZ.js:714:11136
    at Array.forEach (<anonymous>)
    at w4.setLevelFilter (angular-WCVVIDOZ.js:714:11022)
    at Object.next (angular-WCVVIDOZ.js:714:15280)
    at e.next (chunk-UG3CQJHX.js:3:3966)
```

--- a/src/components-examples/journey-maps/angular/journey-maps-full/journey-maps-full-example.ts
+++ b/src/components-examples/journey-maps/angular/journey-maps-full/journey-maps-full-example.ts
@@ -191,7 +191,7 @@ export class JourneyMapsFullExample implements OnInit, OnDestroy {
     this.form = _fb.group({
       mapVisible: [true],
       limitMaxBounds: [false],
-      level: [0],
+      level: [undefined],
       uiOptions: _fb.group({
         showSmallButtons: [false],
         levelSwitch: [true],

--- a/src/journey-maps/angular/components/level-switch/services/level-switcher.ts
+++ b/src/journey-maps/angular/components/level-switch/services/level-switcher.ts
@@ -119,7 +119,7 @@ export class SbbLevelSwitcher {
       });
 
     this._selectedLevel.pipe(takeUntil(this._destroyed)).subscribe((selectedLevel) => {
-      this._mapLayerFilterService.setLevelFilter(selectedLevel);
+      this._mapLayerFilterService.setLevelFilter(selectedLevel ?? 0);
       this._mapTransferService.updateOutdoorWalkFloor(this._map, selectedLevel);
       // call outside component-zone, trigger detect changes manually
       this.changeDetectionEmitter.emit();

--- a/src/journey-maps/angular/components/level-switch/services/map-layer-filter.ts
+++ b/src/journey-maps/angular/components/level-switch/services/map-layer-filter.ts
@@ -17,7 +17,7 @@ export class SbbMapLayerFilter {
   ["==", "floor", 0]
   ["!=", "floor", 0] (It's a compound predicate because of NOT)
    */
-  setLevelFilter(level: number | undefined): void {
+  setLevelFilter(level: number): void {
     this._knownLvlLayerIds.forEach((layerId) => {
       try {
         const oldFilter = this._map.getFilter(layerId) ?? undefined;
@@ -28,7 +28,7 @@ export class SbbMapLayerFilter {
       }
     });
 
-    this._setLayerVisibility('rokas_background_mask', level !== undefined && level < 0);
+    this._setLayerVisibility('rokas_background_mask', level < 0);
   }
 
   private _calculateLayerFilter(

